go/tools/releaser/github.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2021 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/google/go-github/v36/github"
+)
+
+func githubListTags(ctx context.Context, gh *github.Client, org, repo string) (_ []*github.RepositoryTag, err error) {
```

Comment:
Have you considered creating a struct that has a `*github.Client` on it and turning this into just `releaserGH.listTags(ctx, org, repo)`?